jms/1.7/modules/ROOT/pages/jms-listener.adoc

```diff
@@ -1,43 +1,76 @@
-= To Listen For New Messages
+= Listen New Messages Using the JMS Connector Examples
 :keywords: jms, connector, consume, message, source, listener
 :page-aliases: connectors::jms/jms-listener.adoc
 
+Anypoint Connector for JMS (JMS Connector) *On New Message* source enables you to listen and consume new messages as they arrive to the specified destination.
 
+== Listen for New Messages from a Queue
 
-The On New Message (Listener) source in the JMS connector provides ability to consume Messages as they arrive to the Destination.
+In the following example, you configure the *On New Message* source to listen for new messages in the queue specified in the *Destination* field. The operation returns a Mule Message each time a JMS message is available in the queue. The Mule Message has:
 
-== Listening for New Messages
-The syntax to listen for new messages from a queue is:
+* The message's content as payload
+* The message's metadata in the message attributes
+
+By default, the message consumed will be acknowledged (ACKed) only when the execution of the flow receiving the message completes successfully. If instead, an error occurs during the execution of the flow, the session recovers, and the message is redelivered.
```

Comment:
```suggestion
By default, the message consumed is acknowledged ("ACKed") only when the execution of the flow receiving the message completes successfully. If not, the session recovers and the message is redelivered.
```
Do we have to document "ACKed"? We want to avoid turning syntax values into common verbs....